CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required()
 
 project(QuantLib)
 
+if(NOT CMAKE_CXX_STANDARD)
```

Comment:
Also, I would do this just for Mac OS — on Windows and most Linux distribution, the compilers default to a later version of the standard and this would be a step backwards.

Reply:
```
if(APPLE AND NOT CMAKE_CXX_STANDARD)
```
maybe?

Reply:
Yeah sounds good